src/shared-components/Regular1/index.tsx

```diff
@@ -1,14 +1,16 @@
-import { Typography } from '@mui/material';
+import { Typography, SxProps } from '@mui/material';
+import { Theme } from '@mui/material/styles';
+
 interface Regular1Props {
   children: string | JSX.Element;
+  style?: React.CSSProperties;
+  sx?: SxProps<Theme>;
+  id?: string;
 }
 
-export const Regular1 = ({ children }: Regular1Props) => {
+export const Regular1 = ({ children, style = {}, sx, id }: Regular1Props) => {
   return (
-    <Typography
-      variant='body1'
-      sx={{ opacity: '50%', textAlign: 'center', margin: '10px' }}
-    >
+    <Typography id={id} style={style} variant='body1' sx={sx}>
```

Comment:
I would not recommend to make typography components customizable like this. The goal here is that we have stable (=non-changeable) component to use everywhere. I think you can only have children as props here, maybe color if it's needed somewhere.